packages/metrics-scala/metrics/src/main/scala/mutationtesting/MetricsResult.scala

```diff
@@ -59,7 +62,7 @@ sealed trait MetricsResult {
 
   /** All mutants.
     */
-  lazy val totalMutants: Int = totalValid + totalInvalid + ignored
+  lazy val totalMutants: Int = totalValid + totalInvalid + ignored + pending
```

Comment:
pending should probably be added to totalValid instead of to totalMutants directly